Sources/Protocols/MessagesLayoutDelegate.swift

```diff
@@ -26,7 +26,7 @@ import Foundation
 
 /// A protocol used by the `MessagesCollectionViewFlowLayout` object to determine
 /// the size and layout of a `MessageCollectionViewCell` and its contents.
-public protocol MessagesLayoutDelegate: class {
+public protocol MessagesLayoutDelegate: AnyObject {
```

Comment:
@zhongwuzw Why did we change these from `class` -> `AnyObject`? I think a `class` meets the minimum requirement? The reason it's constrained to classes is to have a weak reference

Reply:
@SD10 , yeap, class-only protocol keyword has changed from `class` to `AnyObject` in `Swift4`. [Class-Only Protocols](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Protocols.html#//apple_ref/doc/uid/TP40014097-CH25-ID281)